weaviate/data/crud_data.py

```diff
@@ -700,7 +700,7 @@ def _get_params(additional_properties: Optional[List[str]], with_vector: bool) -
     if additional_properties:
         if not isinstance(additional_properties, list):
             raise TypeError("Additional properties must be of type list "
-                f"but are {type(additional_properties)}")
+                            f"but are {type(additional_properties)}")
```

Comment:
I do not see the need for this many tabs, it does not make it more readable. I would suggest doing this instead:
```python
raise TypeError(
    "Additional properties must be of type list "
    f"but are {type(additional_properties)}"
)
```
or, if line not longer than 100 characters:
```python
raise TypeError(
    f"Additional properties must be of type list but are {type(additional_properties)}"
)
```

Reply:
Ok 👍